client/src/components/Versions/VersionsTable/VersionsTable.js

```diff
@@ -40,96 +31,100 @@ const VersionsTable = ({
     versions,
     renderDeleteVersionButton,
     showDownloadCount,
-    appId,
+    changelogData,
 }) => {
     const getDownloadUrl = useCreateGetDownloadUrl()
-    const [changelogVisible, setChangelogVisible] = useState(false)
-    const showChangeLog = () => {
-        setChangelogVisible(!changelogVisible)
-    }
-
-    const onCloseChangelog = () => {
-        setChangelogVisible(false)
-    }
 
     return (
-        <>
-            {changelogVisible && (
-                <ChangeLogViewer
-                    appId={appId}
-                    onCloseChangelog={onCloseChangelog}
-                />
-            )}
-            <Table className={styles.table}>
-                <TableHead>
-                    <TableRowHead>
-                        <TableCellHead>Version</TableCellHead>
-                        <TableCellHead>Channel</TableCellHead>
-                        <TableCellHead>
-                            DHIS2 version compatibility
-                        </TableCellHead>
-                        <TableCellHead>Upload date</TableCellHead>
-                        {showDownloadCount && (
-                            <TableCellHead>Downloads</TableCellHead>
-                        )}
-                        <TableCellHead></TableCellHead>
-                    </TableRowHead>
-                </TableHead>
-                <TableBody>
-                    {versions.map((version) => (
-                        <TableRow key={version.id}>
-                            <TableCell>{version.version}</TableCell>
-                            <TableCell className={styles.channelNameCell}>
+        <ol className={styles.versionList}>
+            {versions.map((version) => {
+                const changes = changelogData?.[version.version] ?? []
```

Comment:
I was a bit surprised to see this as an array... would it not be better to store / display just a markdown text block for each version change as GitHub releases does?

Reply:
it was like that initially to keep the changes in a structured format and display them consistently, to do things like:
- we were displaying the change type in a cell in a table so this was better than doing regex parsing at last minute
- renaming the change types: "feat" to "feature" etc..
- highlighting (or ignoring) translations
- being able to show only bugs or features, or group by change type (never did that but thought it makes sense when we're showing the changes vs instlaled version for example that we'd show all features then all bugs..)

slowly though, the design evolved, and we're not doing any of these .. so I could go back to just having an array of versions, and the changes as a markdown that we display as it is

Reply:
@amcgee updated it to just show the raw markdown:

![image](https://github.com/user-attachments/assets/2ad75ed7-aa35-4d42-8e49-7f3410e9c2dc)